packages/hint/src/lib/types/jsdom-async-html.ts

```diff
@@ -63,11 +63,16 @@ export class JSDOMAsyncHTMLElement implements IAsyncHTMLElement {
     /* istanbul ignore next */
     public getLocation(): ProblemLocation {
         try {
-            const location = this._dom && this._dom.nodeLocation(this._htmlelement);
+            /*
+             * TODO: Depending on the install (yarn vs npm) we get a different version of `jsdom`
+             * that has a different version of `parse5`. This takes care of this issue but should
+             * be fixed once we migrate completely to jsdom 12 (https://github.com/webhintio/hint/pull/1274)
+             */
+            const location: any = this._dom && this._dom.nodeLocation(this._htmlelement);
```

Comment:
Minor nit since this should be a temporary workaround, but I'd suggest pushing the explicit use of `any` as low as possible so we still get types and intellisense for supported properties. How about an explicit cast on lines 74 and 75?

```typescript
column: location.startTag.col || (location.startTag as any).startCol,
line: (location.startTag.line || (location.startTag as any).startLine) - 1
```